<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slotsgame</class>
 <widget class="QDialog" name="slotsgame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QFrame" name="slotW_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>-70</y>
      <width>100</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/slots/slot.png);</string>
    </property>
   </widget>
   <widget class="QFrame" name="slotW_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>-270</y>
      <width>100</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/slots/slot.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/images/slots/background.png)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: transparent;
image: url(:/images/slots/button.png);
}
QPushButton:pressed {
background-color: transparent;
color: white;    
border: 2px solid transparent;
}
QPushButton:hover{
background-color: transparent;
image: url(:/images/slots/button.png);
}
</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="iconSize">
     <size>
      <width>267</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="slotW">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>-70</y>
      <width>100</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/slots/slot.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>balance:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>bet:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bonus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 25pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>| | | | | bonus | | | | |</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="hidden" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="youWin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 30pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>You Win</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="hidden" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="betValue">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>15</pointsize>
      <weight>87</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="balanceValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="youWin_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 30pt &quot;Magneto&quot;;
color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>You will Win</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>slotW</zorder>
   <zorder>slotW_2</zorder>
   <zorder>slotW_3</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton</zorder>
   <zorder>balance</zorder>
   <zorder>bet</zorder>
   <zorder>bonus</zorder>
   <zorder>youWin</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>betValue</zorder>
   <zorder>balanceValue</zorder>
   <zorder>youWin_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
